### DISCUSSION TITLE
Interpretation of the outputs

### DISCUSSION THREAD

**Discussion by huansouza** [FeynCalc/feyncalc#79] [Q&A] [answered]
Hi,

I'm using FeynCalc with TARCER and when I use the function TarcerExpand, the outputs are a bit strange to me,

![Screenshot_20201105_121414](https://user-images.githubusercontent.com/62699121/98259216-99a54a00-1f60-11eb-8b93-f0a7b7846e27.png)

In the first case, I used the external momentum outside the TarcerExpand function and I obtained a clean result. In the second case, I used the external momentum inside the TarcerExpand function and the result is very different. Besides that, there are some expressions that I never saw before, like Momentum^{(1,0)} (p,4).

My question is, what is the correct form of use this function and if it is the second one, what does mean these expressions like Momentum^{(1,0)} (p,4)?

Cheers,

Huan.

**Comment by vsht** [comment, marked as answer]
As you can see, `TarcerExpand` is not designed to expand amplitudes, only integrals. When you give it an amplitude, it
tries to apply Series to all occurrences of D, which creates such weird output:

```
Series[FCI@SPD[a, b] /. D -> 4 - 2 ep, {ep, 0, 2}]
```

The correct way would be to apply `TarcerExpand` only to integrals. Just use FCLoopExtract as explained in 
arXiv:1601.01167.

**Reply by huansouza** [reply]
I didn't know about that, it worked properly. Thank you again!
